rest/admin_api_test.go

```diff
@@ -290,24 +291,59 @@ function(doc, oldDoc) {
 	input = input + `]}`
 	response = rt.SendAdminRequest("POST", "/db/_bulk_docs", input)
 
+
 	// Start changes feed
 	var wg sync.WaitGroup
+
+	wg.Add(1)
+
+	numExpectedChanges := 201
+
 	go func() {
 		defer wg.Done()
-		wg.Add(1)
-		// Timeout allows us to read continuous changes after processing is complete.  Needs to be long enough to
-		// ensure it doesn't terminate before the first change is sent.
-		changesResponse := rt.Send(requestByUser("GET", "/db/_changes?feed=continuous&since=0&timeout=2000", "", "bernard"))
 
-		changes, err := readContinuousChanges(changesResponse)
-		assert.Equals(t, err, nil)
+		var since uint64
+		since = 0
+
+		maxTries := 10
+		numTries := 0
+
+		changesAccumulated := []db.ChangeEntry{}
+
+		for {
+
+			// Timeout allows us to read continuous changes after processing is complete.  Needs to be long enough to
+			// ensure it doesn't terminate before the first change is sent.
+			changesResponse := rt.Send(requestByUser("GET", fmt.Sprintf("/db/_changes?feed=continuous&since=%d&timeout=5000", since), "", "bernard"))
+
+			changes, err := readContinuousChanges(changesResponse)
+			assert.Equals(t, err, nil)
+
+			changesAccumulated = append(changesAccumulated, changes...)
+
+			if len(changesAccumulated) >= numExpectedChanges {
+				log.Printf("Got numExpectedChanges (%d).  Done", numExpectedChanges)
+				break
+			}
+
+			// Advance the since value if we got any changes
+			if len(changes) > 0 {
+				since = changes[len(changes) - 1].Seq.Seq
```

Comment:
Is it safe to use Seq.Seq here?  That will ignore any lowSequence or triggered by handling.  Using Seq.String() would be safer, but if you know this test won't ever do backfill or have skipped sequences, it might be ok.

Reply:
Good catch, I forgot about compound sequences.  Changed to string.